src/index.js

```diff
@@ -87,7 +92,11 @@ export default class List extends React.Component {
   };
 
   renderList = () => (
-    <div style={style.container} ref={c => (this.container = c)}>
+    <div
+      style={style.container(this.getDefaultHeightWidth())}
+      className={this.props.className ? this.props.className : ''}
```

Comment:
We can just keep the class name as follows:
```js
className={this.props.className}
```
If the prop is not set, the value will be `undefined` and the className key will not be created. However if we use the ternary check, an empty classname will be set when classname prop is not passed.